planetiler-core/src/main/java/com/onthegomap/planetiler/Planetiler.java

```diff
@@ -573,6 +576,11 @@ public Planetiler fetchWikidataNameTranslations(Path defaultWikidataCache) {
         fetchWikidata);
     useWikidata = fetchWikidata || arguments.getBoolean("use_wikidata", "use wikidata translations", true);
     wikidataNamesFile = arguments.file("wikidata_cache", "wikidata cache file", defaultWikidataCache);
+    wikidataMaxAge =
+      arguments.getDuration("wikidata_max_age", "Maximum age of Wikidata translations (0 = disabled)", "720H");
```

Comment:
This method does a naive conversion from string to duration:

```java
Duration parsed = Duration.parse("PT" + value);
```

duration an also handle "30D" except the string needs to be "P30D" or (a full example would be "P2DT3H4M"). If you want you could also update that method to do this conversion to make it a bit easier to use (and maybe call toUpperCase automatically to handle "30d") Or I could do that in a separate PR

Reply:
Adjusted. `Duration.parse()` tweaked to be "backward compatible", so that values like `10s` still work.